src/sentry/static/sentry/app/views/eventsV2/eventView.tsx

```diff
@@ -572,28 +569,41 @@ class EventView {
     });
   }
 
+  withColumns(columns: Column[]): EventView {
```

Comment:
@markstory Updating a column that is sorted will lead to an event view with the stale `sort` query string value. This would need to be accounted for.

Reply:
@markstory also deleting a sorted column persists the old sort value.